lib/managers/files.js

```diff
@@ -1344,6 +1375,53 @@ Files.prototype.getRepresentationInfo = function(fileID, representationType, cal
 		})
 		.asCallback(callback);
 };
+
+/**
+ * Get the contents of a representation of a file, e.g, the binary content of an image or pdf.
+ *
+ * API Endpoint: '/files/:fileID?fields=representations'
+ * Method : GET
+ *
+ * @param {string} fileID The file ID to get the representation of
+ * @param {string} representationType The X-Rep-Hints type to request
+ * @param {Object} [options] Optional parameters
+ * @param {string} [options.assetPath] Asset path for representations with multiple files
+ * @param {Function} [callback] Passed a stream over the representation contents if successful
+ * @returns {Promise<Readable>} A promise resolving to a stream over the representation contents
+ */
+Files.prototype.getRepresentationContent = function(fileID, representationType, options, callback) {
```

Comment:
It's possible to use a regular expression to validate that `representationType` is valid and will only ask for a single representation to be returned (i.e. guarantees 0 or 1 results). If regexes are easy enough in javascript and you'd like to do that, let me know and I'll give you a regex for it (once I figure out what syntax javascript uses).

Reply:
We don't generally do any client-side validation to avoid needing to stay in lock-step with the API and to reduce our surface area for bugs (we don't want to accidentally reject valid input).  We treat the API as the source of truth for what input is valid and rely on it to return an error if the user passes invalid input.  All adding a regex would do is let the user avoid making an API call to see the error; I'm not sure it's worth it, especially since the X-Rep-Hints value is likely developer-provided.

Reply:
That's fine, I just wanted to give the option.